resources/assets/config.json

```diff
@@ -9,7 +9,7 @@
     ]
   },
   "publicPath": "/app/themes/sage",
-  "devUrl": "http://lab.tinypixel.test",
+  "devUrl": "http://example.com",
```

Comment:
Hey Kelly, not sure if the best practice is to comment here on this commit, but I thought I'd let you know this is a .com instead of a .test. Not that anyone's going to use the default value anyway, but thought you might want to know.

Thanks for putting all the work into this PR - looking forward to having this available in Sage.

Reply:
https://en.wikipedia.org/wiki/Example.com 😉

Reply:
Thanks for the catch! 

I should probably just add example.test to /etc/hosts and save myself future embarrassment. 😅

Reply:
@garrettw Good point. Although my laziness is saying that not having to change `.com` to `.test` every time I customize for a project sounds nice. ;)